pom.xml

```diff
@@ -196,6 +196,17 @@
         </dependencies>
     </dependencyManagement>
 
+    <repositories>
```

Comment:
This is related to this issue? Why was added here?

Reply:
So that this builds with the Fabric8 snapshot version.